HousingManagementSystemApi/serverless.yml

```diff
@@ -31,6 +31,8 @@ functions:
     environment:
       AUTHENTICATION_IDENTIFIER: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/authentication-identifier}
       JWT_SECRET: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/jwt-secret}
+      HOUSING_SEARCH_API_URI: ${ssm:/HousingManagementSystemApi/${self:provider.stage}/housing-search-api-uri
+      HOUSING_SEARCH_API_KEY:  ${ssm:/HousingManagementSystemApi/${self:provider.stage}/housing-search-api-key
     events:
```

Comment:
Missing env vars for the properties uri and key

Reply:
Added